charmcraft.yaml

```diff
@@ -13,4 +13,10 @@ bases:
 parts:
   charm:
     charm-binary-python-packages:
+      - cryptography
```

Comment:
No need to add this here since we added `rustc` and the others, we should be able to build it.